lib/application.js

```diff
@@ -455,6 +455,17 @@ app._verifyAuthModelRelations = function() {
 
   function verifyUserRelations(Model) {
     const hasManyTokens = Model.relations && Model.relations.accessTokens;
+
+    // display a temp warning message for users using multiple users config
+    if (hasManyTokens.polymorphic) {
+      console.warn(
+        'The app configuration follows the multiple user models setup ' +
+          'as described in http://ibm.biz/setup-loopback-auth',
+          'The built-in role resolver $owner is not currently compatible ' +
+          'with this configuration and should not be used in production.',
+        Model.modelName);
+    }
+
```

Comment:
i added a generic warning msg that pops anytime a User model is configured with multiple-users support. This is detected with the existence of a polymorphic relation from AccessToken to users.

The message covers a bit large, but this was the easiest way to implement rather than also checking if a model has also ownerRelation option configured. Plus in some edge cases, the legacy behavior could be impacted too.

-> Will submit the fix in a new PR shortly.